File: BAEKJOON/java/26068_치킨댄스를_추는_곰곰이를_본_임스_2.java
import java.io.*;
import java.util.ArrayList;
import java.util.List;

public class Main {
    public static int n, answer = 0;
    public static List<String> list = new ArrayList<>();

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        n = Integer.parseInt(br.readLine());

        for (int i = 0; i < n; i++) {
            String s = br.readLine();
            list.add(s);
        }
    }

    private static void solution() throws IOException {
        StringBuilder sb;

        for (int i = 0; i < list.size(); i++) {
            sb = new StringBuilder(list.get(i));
            sb.delete(0, 2);

            int day = Integer.parseInt(sb.toString());
            if (day <= 90) {
                answer++;
            }
        }

        bw.write(String.valueOf(answer));
    }
}

File: BAEKJOON/cpp/26068_치킨댄스를_추는_곰곰이를_본_임스_2.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int n, answer = 0;
vector<string> v;

void input() {
	cin >> n;

	string s;
	for (int i = 0; i < n; i++) {
		cin >> s;
		v.push_back(s);
	}
}

void solution() {
	string s;
	for (int i = 0; i < v.size(); i++) {
		s = v[i];
		s.erase(s.begin(), s.begin() + 2);

		int day = stoi(s);
		if (day <= 90) {
			answer++;
		}
	}

	cout << answer;
}

int main()
{
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}